source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTableCellRenderer.mm

```diff
@@ -20,4 +26,41 @@ + (ACRCardElementType)elemType
     return ACRTableCell;
 }
 
+
+- (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
+           rootView:(ACRView *)rootView
+             inputs:(NSMutableArray *)inputs
+    baseCardElement:(ACOBaseCardElement *)acoElem
+         hostConfig:(ACOHostConfig *)acoConfig;
+{
+    std::shared_ptr<BaseCardElement> elem = [acoElem element];
+    auto cellElement = std::dynamic_pointer_cast<TableCell>(elem);
+
+    ACRColumnView *cell = (ACRColumnView *)viewGroup;
+
+    cell.rtl = rootView.context.rtl;
+
+    renderBackgroundImage(cellElement->GetBackgroundImage(), cell, rootView);
+
+    [ACRRenderer render:cell
+               rootView:rootView
+                 inputs:inputs
+          withCardElems:cellElement->GetItems()
+          andHostConfig:acoConfig];
+
+    [cell setClipsToBounds:NO];
+
+    std::shared_ptr<BaseActionElement> selectAction = cellElement->GetSelectAction();
+    ACOBaseActionElement *acoSelectAction = [ACOBaseActionElement getACOActionElementFromAdaptiveElement:selectAction];
+    [cell configureForSelectAction:acoSelectAction rootView:rootView];
```

Comment:
just so I'm sure: no negative consequences for a null selectaction here?

Reply:
no, the method will do null check.